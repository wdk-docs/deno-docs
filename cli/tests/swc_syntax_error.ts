for await (const req of s) {
    let something:
}
