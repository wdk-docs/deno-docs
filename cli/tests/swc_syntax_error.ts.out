error: Unexpected token Some(RBrace) at [WILDCARD]syntax_error.ts:3:0
